app/screens/auth/CreateWallet.tsx

```diff
@@ -159,36 +139,25 @@ const CreateWallet = ({ history, location }: AuthRouterParams) => {
     setConvenientWalletName(value);
   };
 
-  const nextAction = () => {
+  const nextAction = async () => {
```

Comment:
Not sure exactly where's the issue, didn’t take time to analyse, but: 

during new wallet creation, once you set up a password, skip the mnemonics setup, there's a confirmation screen with a link to the wallet file location. At this point the link opens the Spacemesh directory, but the wallet file is not yet there. It is created later, after you close this confirmation screen and open smapp. it probably should be corrected to avoid "my wallet wasn’t created" kind of reports...

https://github.com/spacemeshos/smapp/assets/89876259/1d0c3fba-43e5-4c27-b175-87d685305082